shrdlurn/csscolors.lexicon

```diff
@@ -0,0 +1,148 @@
+{"lexeme": "AliceBlue", "formula": "aliceblue", "type": "color"}
+{"lexeme": "AntiqueWhite", "formula": "antiquewhite", "type": "color"}
+{"lexeme": "Aqua", "formula": "aqua", "type": "color"}
+{"lexeme": "Aquamarine", "formula": "aquamarine", "type": "color"}
+{"lexeme": "Azure", "formula": "azure", "type": "color"}
+{"lexeme": "Beige", "formula": "beige", "type": "color"}
+{"lexeme": "Bisque", "formula": "bisque", "type": "color"}
+{"lexeme": "Black", "formula": "black", "type": "color"}
+{"lexeme": "BlanchedAlmond", "formula": "blanchedalmond", "type": "color"}
```

Comment:
no, they would not. CSS color seems standard, and they are usually not spaced in the references.